File: Estrutura_Sequencial/JavaScript/Ex1013.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
var prompt = function(texto) {return lines.shift();};

var [A, B, C] = prompt().split(' ').map(Number);

maiorAB = (A + B + Math.abs(A - B)) / 2;
maior = (maiorAB + C + Math.abs(maiorAB - C)) / 2;

console.log(maior, "eh o maior");

File: Estrutura_Sequencial/C#/Ex1013.cs
using System;

class Program
{
    static void Main(string[] args)
    {
        string[] valores = Console.ReadLine().Split();
        int a = int.Parse(valores[0]);
        int b = int.Parse(valores[1]);
        int c = int.Parse(valores[2]);

        int maiorAB = (a + b + Math.Abs(a - b)) / 2;

        int maior = (maiorAB + c + Math.Abs(maiorAB - c)) / 2;

        Console.WriteLine($"{maior} eh o maior");
    }
}